panther_analysis_tool/schemas.py

```diff
@@ -193,6 +193,33 @@ def validate(
     ignore_extra_keys=False,
 )  # Prevent user typos on optional fields
 
+DERIVED_SCHEMA = Schema(
+    {
+        "AnalysisType": Or("rule", "scheduled_rule"),
+        "RuleID": And(str, NAME_ID_VALIDATION_REGEX),
+        "BaseDetection": And(str, NAME_ID_VALIDATION_REGEX),
+        Optional("Enabled"): bool,
+        Optional("Severity"): Or("Info", "Low", "Medium", "High", "Critical"),
+        Optional("Description"): str,
+        Optional("DedupPeriodMinutes"): int,
+        Optional("InlineFilters"): object,
+        Optional("DisplayName"): And(str, NAME_ID_VALIDATION_REGEX),
+        Optional("OnlyUseBaseRiskScore"): bool,
+        Optional("OutputIds"): [str],
+        Optional("Reference"): str,
+        Optional("Runbook"): str,
+        Optional("SummaryAttributes"): [str],
+        Optional("Threshold"): int,
+        Optional("Tags"): [str],
+        Optional("Reports"): {str: list},
+        Optional("DynamicSeverities"): object,
+        Optional("AlertTitle"): str,
+        Optional("AlertContext"): object,
+        Optional("GroupBy"): object,
+    },
+    ignore_extra_keys=False,
```

Comment:
I don't see this being used anywhere I think you'll need to update [this section](https://github.com/panther-labs/panther_analysis_tool/blob/6b45e21d132e3a0d59029d4509d6e920984c5a43/panther_analysis_tool/main.py#L2030) to include the derived schema and set to true conditionally as we seem to do for other types

Reply:
Added, thanks.